docs/workflow/blocks/form.rst

```diff
@@ -157,4 +157,190 @@ In the example below, a mapping has a default value, which is saved using the co
         },
         "uiSchema": {}
       }
-    ]
\ No newline at end of file
+  ]
+
+Password Fields
+---------------
+
+You can use uiSchema to designate a password field. 
+The below example will render a form that displays only * for each character entered in the "password" field.
+
+You can see this working `here
+<https://app.kendra.io/bandsintown/connect>`_.
```

Comment:
:clap: :clap: :clap:  I love that you add an example flow! 
Which also means we have to keep the flow update and running

Reply:
@gsambrotta I changed it to a share link, which should mitigate against at least some future breaking changes.